profiles/seccomp/seccomp.go

```diff
@@ -89,25 +100,25 @@ Loop:
 	// Loop through all syscall blocks and convert them to libcontainer format after filtering them
 	for _, call := range config.Syscalls {
 		if len(call.Excludes.Arches) > 0 {
-			if stringutils.InSlice(call.Excludes.Arches, arch) {
+			if inSlice(call.Excludes.Arches, arch) {
```

Comment:
Splitting this utility to the various parts where we use it is probably a good opportunity to check if these should all be case-insensitive as well.

ping @justincormack could you have a look to see if these should all be case-insensitive?

Reply:
technically this would be an API change if we changed the case sensitivity, although it never needed to be insensitive.